src/ansys/openapi/common/_api_client.py

```diff
@@ -362,11 +365,18 @@ def __deserialize(self, data: SerializedType, klass_name: str) -> DeserializedTy
 
             * String class name
             * String type definition for list or dictionary
+            * "object" literal, which returns the dictionary as-is
         """
 
         if data is None:
             return None
 
+        if klass_name == "object":
+            warnings.warn("Attempting to deserialize an object with no defined type. Returning "
```

Comment:
I wanted to provide more information about the problematic model, but it's surprisingly hard to get to given the recursive calling.